packages/fxa-support-panel/lib/api.ts

```diff
@@ -156,6 +157,7 @@ class SupportController {
 
     const formattedSubscriptions = subscriptions.map(s => ({
       ...s,
+      created: String(new Date(s.created * MS_IN_SEC)),
```

Comment:
Are there tests that assert that the page elements exists?

Reply:
There are some existing tests for some elements, but not all of them.

Reply:
> @jackiemunroe I wasn't able to test this locally, maybe something with my setup?
> 
> <img alt="Screen Shot 2020-03-12 at 11 20 09 AM" width="991" src="https://user-images.githubusercontent.com/1295288/76536985-7c60c000-6453-11ea-8690-37fb92d3c90b.png">

Yeah, it was a bit of a journey to get it working locally for myself. I'm going to add a test to ensure the element exists

Reply:
Ah ok, I'll defer on trying to debug my local setup for now :)